### PR TITLE
Added another installation client for preselecting cloud admin pattern for installation

### PR DESCRIPTION
**Pull request by jsuchome**
(based on recent product metadata, it is not selected automatically)

### CODE REVIEW

File: src/clients/inst_crowbar_patterns.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Authors:     Jiri Suchomel <jsuchome@suse.cz>
+#
+module Yast
+  class InstCrowbarPatternsClient < Client
+    def main
+
+      Yast.import "GetInstArgs"
+      Yast.import "PackagesProposal"
+
+      @dialog_ret = :auto
```
Comment (lslezak) [inline src/clients/inst_crowbar_patterns.rb]: Just nit picking: If you do not need to store the values in the object attributes use simple `dialog_ret` and `patterns` variables.

**Comment by lslezak** [comment]
Otherwise LGTM

File: src/clients/inst_crowbar_patterns.rb (hunk 1 above)
Comment (jsuchome) [inline src/clients/inst_crowbar_patterns.rb]: True

**Comment by lslezak** [comment]
LGTM, thanks

**Comment by vuntz** [comment]
@jsuchome feel free to merge if that works, I don't think anyone else in the cloud team can give a proper +1 ;-)

**Pull request merged**